src/test/java/org/junitpioneer/jupiter/DisableIfTestFailsTests.java

```diff
@@ -10,7 +10,7 @@
 
 package org.junitpioneer.jupiter;
 
-import static org.junit.jupiter.api.Assertions.fail;
```

Comment:
If we want no JUpiter assertions at all, maybe ban the import ([here](https://github.com/junit-pioneer/junit-pioneer/blob/main/.infra/checkstyle/import-control.xml))?